app/react/src/client/preview/index.js

```diff
@@ -20,7 +20,13 @@ const isBrowser =
   !(navigator.userAgent.indexOf('Node.js') > -1);
 
 const storyStore = new StoryStore();
-const reduxStore = createStore(reducer);
+/* eslint-disable no-underscore-dangle */
+const reduxStore = createStore(
+  reducer,
+  window.__REDUX_DEVTOOLS_EXTENSION__ &&
+    window.__REDUX_DEVTOOLS_EXTENSION__({ name: 'Storybook Preview', instanceId: 'sbPreview' })
+);
+/* eslint-enable */
```

Comment:
What's happening with this change? It seems unrelated.

Reply:
Correct, it is unrelated. It's enabling dev tools. Do you think it should be in a separate branch/pr.

Reply:
I think yeah, let's move to a separate PR